plugins/nf-validation/src/main/nextflow/validation/SchemaValidator.groovy

```diff
@@ -515,7 +515,7 @@ class SchemaValidator extends PluginExtensionPoint {
             for (int i=0; i < arrayJSON.size(); i++) {
                 def JSONObject entry = arrayJSON.getJSONObject(i)
                 if ( entry.has(filedName) ) {
-                    pathExists(entry[filedName].toString())
+                    pathExists(entry[filedName].toString(), " Entry ${(i+1).toString()} - ${fieldName.toString()}")
```

Comment:
Why are we doing this?

Having two variables that look so similar seems like a recipe for disaster 😬 

```groovy
def String filedName = fieldName
```

Reply:
That was probably one of my typing fast typos, thanks for spotting it! 😄  we only need to declare the variable as a string